doc_improvemnts:Added commands for Apple M series chip (#436)
Parallels/vagrant-parallels: website/docs/source/docs/boxes/index.html.md

All boxes for "parallels" provider could be found on Vagrant Cloud:
[https://app.vagrantup.com/boxes/search?provider=parallels](https://app.vagrantup.com/boxes/search?provider=parallels)

> For Apple M series chip use arm based boxes
> [https://app.vagrantup.com/boxes/search?utf8=%E2%9C%93&sort=downloads&provider=parallels&q=arm](https://app.vagrantup.com/boxes/search?utf8=%E2%9C%93&sort=downloads&provider=parallels&q=arm)

Reviewer: **suggestion:** Let's rephrase it this way?
```suggestion
For Macs with Apple M-series chip use ARM-based boxes:
[https://app.vagrantup.com/boxes/search?provider=parallels&q=arm](https://app.vagrantup.com/boxes/search?provider=parallels&q=arm)
```

You can also quickly initialize a Vagrant environment with the command:

> For Intel chip use

Reviewer: ```suggestion
For Macs with Intel chip:
```

```
$ vagrant init bento/ubuntu-18.04
...
```

> For Apple M series chip use

Reviewer: **nitpick:** Let's use the consistent wording "Macs with Apple M-series chip" :
```suggestion
For Macs with Apple M-series chip:
```

```

> $ vagrant init bento/ubuntu-20.04-arm64 \
>   --box-version 202112.19.0
> $ ...

Reviewer: **nitpick:** I guess, the box version is not necessary in the common use case. 
```suggestion
$ vagrant init bento/ubuntu-20.04-arm64
...
```
